brainstormforce/astra
# AST-3349 - Fix: Spectra Compatibility CSS which overrides Spectra Block Spacing Removed.

### Description

- Earlier, during our Spectra Compatibility Release, in order to match up Frontend and Editor, we had written CSS Specific to Spectra blocks to align with Editor Page Title, but it is now causing issues with the Block. As Block Specific CSS is not overriding the Theme CSS. Therefore, we need to remove the CSS.
- JIRA : https://brainstormforce.atlassian.net/browse/AST-3349

### Screenshots

- [Screen Capture on 2023-07-25 at 16-29-06.mp4](https://d.pr/v/Pmb6n0)

### Types of changes




- Bug Fixes :- Spectra Block Specific Padding Left Not Overidding as Expected.
- Breaking Change :- Alignment with the Editor Page Title.

### How has this been tested?

- Install Spectra.
- Add a new Post/Page.
- Spectra Block Specific Padding Left should apply correctly as expected in frontend.
- Refer to video :- - [Screen Capture on 2023-07-25 at 16-29-06.mp4](https://d.pr/v/Pmb6n0)

### Checklist:
